examples/odin/default.odin

```diff
@@ -0,0 +1,10 @@
+package foo
+
+foreign {
```

Comment:
We usually have a function that squares a number as the default example. Is there any reason not to have it with Odin?